virtool/http/auth.py

```diff
@@ -117,7 +116,7 @@ async def authenticate_with_b2c(req: Request, handler: Callable) -> Response:
 
     try:
         token_claims = await validate_token(req.app, token)
-    except (JWTClaimsError, JWTError, ExpiredSignatureError):
+    except JWTError:
```

Comment:
Why? Did you actually test this with B2C?

Reply:
Deepsource, no. Reverted. Deepsource is probably right, but I am not convinced that it is a good idea to merge without being able to test it.

Reply:
Oh. I guess the two removed errors are subclasses of `JWTError`. Let's just leave it.